### DISCUSSION TITLE
Save object class discord.Member

### DISCUSSION THREAD

**Discussion by Vicrorege** [Rapptz/discord.py#9896] [Q&A]
Please tell me how I can send a message to a specific user in PM? As I understand, in order to send a message in PM I must have an object of the class "discord.Member". But if I start a new session, all variables are reset. The question is: can I somehow save an object of the class "discord.Member", for example, to a file? Thanks in advance!

**Comment by DA-344** [comment]
To send DMs to a user to donnot need a `Member`, with only their ID works.
Use [`Client.create_dm`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Client.create_dm) to create or get a DM channel from the specified user, then use that as a normal channel.

**Reply by LeoCx1000** [reply]
You can also use the short form “user.send” - but to answer the original question, you can use “client.get_user” to find a specific user via ID after the bot is ready. This requires the members intent, if you do not have it, then use “client.fetch_user”.

**Reply by MCausc78** [reply]
You probably don't need have user's cache. Like said before, you can use `create_dm`. [Internally](https://github.com/Rapptz/discord.py/blob/ff638d393d0f5a83639ccc087bec9bf588b59a22/discord/client.py#L2942-L2948), it will look up in `{user_id -> dm_channel}` cache first to resolve DM channel, and will send API request if not found in cache. Use `discord.Object(12345)` to not fetch user from API/cache.
